pkg/cli/admin/mustgather/mustgather_test.go

```diff
@@ -5,12 +5,17 @@ import (
 	"reflect"
 	"testing"
 
+	configv1 "github.com/openshift/api/config/v1"
```

Comment:
openshift related dependencies should go after k8s as a separate group

Reply:
Done.